### ISSUE TITLE
JToken.Should().BeEquivalentTo(differentJToken) returns exception

### ISSUE THREAD

**Issue by jakubozga** [fluentassertions/fluentassertions.json#7]
Hello,

Lately I found that if you have 2 different JTokens and call `Should().BeEquivalentTo(...)` you will get:
"System.FormatException: Input string was not in a correct format." instead of error message. 

My guess is that this exception is caused because code is trying to call `string.Format(JToken)`. JToken contains characters like `{` and `}` which should be escaped to `{{` and `}}`

**Comment by dennisdoomen** [comment]
Was that `BeEquivalentTo(...)` provided by the FluentAssertions.Json package or the core library?

**Comment by lancerdima** [comment]
Hi,

Decided to try out your library, and hit the same issue reported by @jakubozga .
In my case:
* Library is used in netcoreapp2.0 test project
* I made sure BeEquivalentTo(...) operation is provided by FluentAssertions.Json package, JTokenAssertions class
* I am comparing two JTokens extracted from HttpRequestMessage StringContents

**Comment by dennisdoomen** [comment]
> Could it be that your MessageBuilder class in the original FluentAssertions app is not handling well messages without placeholders?

Yes, there was a problem in 4.19.3. See https://github.com/fluentassertions/fluentassertions/releases/tag/4.19.3

**Comment by lancerdima** [comment]
Using versions:
FluentAssertions - v4.19.4
FluentAssertions.Json - v4.20.0

Issue seems to be away with pre-release 5.0.0-beta0002 and 5.0.0-beta0003.

**Comment by dennisdoomen** [comment]
Hmm, I found the [fix](https://github.com/fluentassertions/fluentassertions/commit/cfa766bca153f375723132fb0cadde5fcae73130). Unfortunately, I can't resolve this issue on 4.x. You'll have to wait for 5.0.

**Comment by 304NotModified** [comment]
Guess this could be closed? (V6 is released)

**Comment by dennisdoomen** [comment]
Not sure. Completely lost track of this one.
